infra/base-images/base-builder/Dockerfile

```diff
@@ -133,14 +133,29 @@ RUN cd $SRC && \
     tar -xzv --strip-components=1 -f $SRC/oss-fuzz.tar.gz && \
     rm -rf examples $SRC/oss-fuzz.tar.gz
 
+RUN cd $SRC && \
+    git clone \
+      --depth 1 \
+      --branch main \
+      --single-branch \
+      'https://github.com/google/centipede.git' \
+      'centipede' && \
+    cd centipede && \
+    echo 'build --client_env=CC=clang --cxxopt=-std=c++17 ' \
```

Comment:
Odd that you need to add this.

Reply:
@Alan32Liu is this necessary given https://github.com/google/centipede/blob/main/.bazelrc ?

i.e. does our .bazelrc append, or replace all options? If it just appends, maybe we can omit this?

Reply:
Yep, fixed : )